backend/src/services/user/user-service.ts

```diff
@@ -80,6 +81,17 @@ export const userServiceFactory = ({
   const verifyEmailVerificationCode = async (username: string, code: string) => {
     // akhilmhdh: case sensitive email resolution
     const usersByusername = await userDAL.findUserByUsername(username);
+
+    logger.info(
+      usersByusername.map((user) => ({
+        id: user.id,
+        email: user.email,
+        username: user.username,
+        isEmailVerified: user.isEmailVerified
+      })),
+      `Verify email users: ${username}`
```

Comment:
nit: to make it easier to query in cloud watch, it's good to follow this pattern: `[username=${username}]`